label_studio/utils/io.py

```diff
@@ -100,7 +100,7 @@ def remove_file_or_dir(path):
 def iter_files(root_dir, ext):
     for root, _, files in os.walk(root_dir):
         for f in files:
-            if f.lower().endswith(ext):
+            if f.lower().endswith(ext) and not f.lower().startswith('.'):
```

Comment:
It's a good idea to avoid tier files, but iter_files is used in another places too. I think it's better to implement startswith check here: https://github.com/imadcat/label-studio/blob/master/label_studio/storage/filesystem.py#L112

Reply:
@imadcat